### ISSUE TITLE
Typo Bug in the docs

### ISSUE THREAD

**Issue by Legorooj** [Legrandin/pycryptodome#355]
The documention says about `PBKDF2`:
## For example, if you need to derive two AES256 keys:
```
from Crypto.Protocol.KDF import PBKDF2
from Crypto.Hash import SHA512
from Crypto.Random import get_random_bytes

password = b'my super secret'
salt = get_random_bytes(16)
keys = PBKDF2(password, salt, 32, count=1000000, hmac_hash_module=SHA512)
key1 = keys[:32]
key2 = keys[32:]
```
The length of the key you are generating is `32` bytes, therefore `key1 = keys[:32], key2 = keys[32:]` are equal to `X*32` and `''`.

**Comment by Legorooj** [comment]
This should be changed to:
```
keys = PBKDF2(password, salt, 64, count=1000000, hmac_hash_module=SHA512)
```

**Comment by Legrandin** [comment]
Fixed with d02727cf0af5341060cda2d7f11403d7094ec5ad

**Comment by Legorooj** [comment]
Glad to have helped
